packages/core/content-manager/admin/src/components/LeftMenu.tsx

```diff
@@ -89,6 +97,38 @@ const LeftMenu = () => {
     defaultMessage: 'Content',
   });
 
+  /**
+   * @description
+   * Removes i18n from the plugins search param if the schema does not have i18n enabled
+   */
+  const getPluginsParamsForLink = (link: ContentManagerLink) => {
+    const schema = schemas.find((schema) => schema.uid === link.uid);
+    const isI18nEnabled = Boolean(
+      schema?.pluginOptions &&
+        'i18n' in schema?.pluginOptions &&
+        // 'localized' is not typed on pluginOptions.i18n so it can't be accessed directly
+        Object.entries(schema.pluginOptions.i18n || {}).filter(
+          ([key, value]) => key === 'localized' && value === true
+        )
```

Comment:
too bad the types are bad, but this seems pretty complex to me 🤔 wouldn't it be more readable to do `as any` here with some optional chaining?

Reply:
Yup that would 1000 times better than this nonsense 🤦‍♂️

Reply:
What's missing exactly on `schema.PluginOptions.i18n`?

Should there be a `localized` attribute?

Reply:
As far as I can tell, yes there should be a localized attribute.